arviz/plots/backends/bokeh/kdeplot.py

```diff
@@ -151,6 +150,11 @@ def plot_kde(
             glyphs.append(line)
 
     else:
+        try:
+            import contourpy
+        except ImportError as err:
+            raise ImportError("Install contourpy package to enable contour plots.") from err
```

Comment:
How did we handle these messages?

Basically contourpy will be installed by the matplotlib (the latests versions) but I added it to the optional requirements.txt too.

Should the message have the `pip install contourpy`?